src/MongoDB.Driver.Core/Core/Misc/SemanticVersion.cs

```diff
@@ -262,13 +319,8 @@ public static bool TryParse(string value, out SemanticVersion result)
         /// </returns>
         public static bool operator >(SemanticVersion a, SemanticVersion b)
         {
-            if (a == null)
+            if (object.ReferenceEquals(a, null))
             {
-                if (b == null)
```

Comment:
nit: not sure how much this change is required. I think there is no real difference

Reply:
It was removed because it was a bug.

If `a` and `b` are both `null` the return value should be `false`.